src/test/system_tests/test_ceph_s3.js

```diff
@@ -17,23 +17,28 @@ const promise_utils = require('../../util/promise_utils');
 require('../../util/dotenv').load();
 
 const {
-    mgmt_ip = '127.0.0.1',
+    os = '',
+        pass = '',
+        protocol = 'ws',
+        mgmt_ip = '127.0.0.1',
         mgmt_port = '8080',
         s3_ip = '127.0.0.1',
+        s3_acc_key = 123,
+        s3_sec_key = 'abc',
 } = argv;
 
 
 const api = require('../../api');
 let rpc = api.new_rpc();
 
 let client = rpc.new_client({
-    address: `ws://${mgmt_ip}:${mgmt_port}`
+    address: `${protocol}://${mgmt_ip}:${mgmt_port}` // http/https and give the mgmt ip and port
 });
 
 let auth_params = {
-    email: 'demo@noobaa.com',
-    password: 'DeMo1',
-    system: 'demo'
+    email: 'admin@noobaa.io',
+    password: `${pass}`,
+    system: 'noobaa'
```

Comment:
When creating a local noobaa, isn't demo the default?

Reply:
And we should also be able to pass the password via the src/test/framework/test_env_builder_kubernetes.js

Reply:
Running locally on mac results the following error: 's3-tests can only be run on Red Hat, Centos, Fedora, Ubunutu, or Debian platforms'.
I can keep it though but I believe that we won't run those tests locally again.
will talk to you on DM about the password issue.